pom.xml

```diff
@@ -28,18 +28,34 @@
         <selenium.version>3.1.0</selenium.version>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.amazonaws</groupId>
+                <artifactId>aws-java-sdk-bom</artifactId>
```

Comment:
The bom, or [bill of materials](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Importing_Dependencies), ensures all dependencies used have the same version, but does not actually bring dependencies into the project. The JARs brought into the project are only those dependent on aws-java-sdk-s3.